internal/app/api/v1/server.go

```diff
@@ -160,6 +160,19 @@ func (s *TestkubeAPI) WithTelemetry(enabled bool) {
 	s.TelemetryEnabled = enabled
 }
 
+// SendTelemetryStartEvent sends anonymous start event to telemetry trackers
+func (s TestkubeAPI) SendTelemetryStartEvent() {
+	if !s.TelemetryEnabled {
+		return
+	}
+
+	out, err := telemetry.SendServerStartEvent(s.Config.ClusterID, api.Version)
+	if err != nil {
+		s.Log.Debug("telemetry send error", "error", err.Error())
+	}
+	s.Log.Debugw("sending telemetry server start event", "output", out)
```

Comment:
Probably, should not be logged in case of error

Reply:
good catch! fixed!